extensions-core/namespace-lookup/pom.xml

```diff
@@ -77,5 +77,10 @@
       <version>3.0.1</version>
```

Comment:
Does it need to? there are other extensions who have extension-specific (aka nowhere else in druid) library versions in their pom. Is there a reason why this one needs to be in parent pom?